File: Domain/Grupo.cs
using System;

namespace Domain
{
    public class Grupo
    {
        public Guid Id { get; set; }
        public string Titulo { get; set; }
        public string SubTitulo { get; set; }
        public string Descricao { get; set; }
        public string Label { get; set; }
    }
}

File: client-app/src/app/models/grupo.ts
export interface IGrupo {
  id: string;
  titulo: string;
  subTitulo: string;
  descricao: string;
  label: string;
}

export interface IGrupoFormValues extends Partial<IGrupo> {
  // FUTURO USO para algo opcional
}

export class GrupoFormValues implements IGrupoFormValues {
  id?: string = undefined;
  titulo: string = "";
  subTitulo: string = "";
  descricao: string = "";
  label: string = "";

  constructor(init?: IGrupoFormValues) {
    Object.assign(this, init);
  }
}
